modules/installation/pages/install-proxy.adoc

```diff
@@ -1,8 +1,123 @@
 [[installation-proxy]]
-= {productname} {productnumber} Proxy
+= Install {productname} Proxy
 
+This chapter provides the required KVM settings for installation of {sles} media as the base for {productname} Proxy.
```

Comment:
```suggestion
To install {productname} Proxy, you will need to start by installing {sles} media.
This section covers the KVM settings required to perform a {sles} installation as the base for {productname} Proxy.
```

I know it's longer, but it gets the main point up front, and then describes the finer details (rather than the other way around).